packages/ibm-products/src/components/CreateTearsheet/CreateTearsheetDivider.tsx

```diff
@@ -1,19 +1,28 @@
 /**
- * Copyright IBM Corp. 2021, 2021
+ * Copyright IBM Corp. 2021, 2024
  *
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
 
-import React, { forwardRef } from 'react';
+import React, { forwardRef, PropsWithChildren } from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 import { pkg } from '../../settings';
 
 const componentName = 'CreateTearsheetDivider';
 const blockClass = `${pkg.prefix}--tearsheet-create__section--divider`;
+
+interface CreateTearsheetDividerProps {
+  /** Specifies an optional className to be added to the tearsheet divider */
+  className?: string;
+}
+
 export let CreateTearsheetDivider = forwardRef(
-  ({ className, ...rest }, ref) => {
+  (
+    { className, ...rest }: PropsWithChildren<CreateTearsheetDividerProps>,
```

Comment:
I think a better option instead of `PropsWithChildren` might be to just use `React.FC<CreateTearsheetDividerProps>` since this component doesn't accept any children.

Reply:
It would look something like this:
```jsx
export let CreateTearsheetDivider: React.FC<CreateTearsheetDividerProps> = forwardRef(
  ({
    className,
    ...rest,
  },
  ref: ForwardedRef<HTMLSpanElement>
)
```